bin/smoke-test.sh

```diff
@@ -528,6 +528,34 @@ requestMacaroon() { # $1 Caveats, $2 URL, $3 variable for macaroon, $4 variable
     eval $3="$macaroon"
 }
 
+requestWlcgToken() { # $1 oidc-agent account name, $2 audience, $3 variable for token, $4 token for result.
```

Comment:
Very minor comment, but would this be better as `requestOidcAgentToken` (or similar)?

There doesn't seem to be anything specific to WLCG-Profile here.

Reply:
Sure, good idea.